dist-persist/wbstack/src/Settings/LocalSettings.php

```diff
@@ -735,4 +735,110 @@ function getElasticClusterConfig( string $prefix ) {
     }
 }
 
+if ($wwIsLocalisationRebuild) {
+    $extensions = [];
+
+    $loadSkin = function ($name) use (&$extensions, $wgBaseDirectory) {
+        array_push($extensions, "$wgBaseDirectory/skins/" . $name . "/skin.json");
+    };
+    $loadExtension = function ($name) use (&$extensions, $wgBaseDirectory) {
+        array_push($extensions, "$wgBaseDirectory/extensions/" . $name . "/extension.json");
+    };
+    $loadPath = function ($path) use (&$extensions) {
+        array_push($extensions, $path);
+    };
+
+    $loadSkin( 'Vector' );
+    $loadSkin( 'Timeless' );
+    $loadSkin( 'Modern' );
+    $loadSkin( 'MinervaNeue' );
+    $loadExtension( 'SyntaxHighlight_GeSHi' );
```

Comment:
Would it be possible that instead of having to call this for every extension that we `wfLoadExtension` again, we wrap this function to both load the extension and also push it onto the list?

I.e.

```
$loadExtension = function ($name) use (&$extensions, $wgBaseDirectory) {
        wfLoadExtension($name);
        array_push($extensions, "$wgBaseDirectory/extensions/" . $name . "/extension.json");
};
```

I do understand we'd still need to cater for the ConfirmEdit special case when doing it like this, but I am a bit worried having to load extensions twice in LocalSettings will create nasty to debug problems at some point in the future.